cpp/src/arrow/engine/substrait/ext_test.cc

```diff
@@ -248,22 +257,22 @@ TEST(ExtensionIdRegistryTest, RegisterNestedFunctions) {
   for (int i = 0; i < rounds; i++) {
     auto registry1 = substrait::MakeExtensionIdRegistry();
 
-    ASSERT_OK(registry1->CanRegisterFunction(id1, name1.to_string()));
-    ASSERT_OK(registry1->RegisterFunction(id1, name1.to_string()));
+    ASSERT_OK(registry1->CanAddSubstraitCallToArrow(id1));
+    ASSERT_OK(registry1->AddSubstraitCallToArrow(id1, name1.to_string()));
```

Comment:
nit: `name1` is this okay? since there is `name2` could it be renamed?

Reply:
I don't understand.  Are you suggesting `name`/`name2` instead of `name1`/`name2`?

Reply:
Ping

Reply:
Ah no I mean instead it could be an array `name[0]` and `name[1]`. It is not a must here.

Reply:
Ah, I see your point.  I was just porting over an existing test and I think both are probably ok so I think I'd prefer to leave it as is.

Reply:
Yes. It’s probably fine for this.